IMU_RAW_TOPIC: "/camera/imu"
IMU_FILTERED_TOPIC: "/kf/imu"
L_MOTOR_RATE_TOPIC: "/speed1"
R_MOTOR_RATE_TOPIC: "/speed2"
FILTER_TYPE: "ukf" # Either "ukf" or "ekf"

KF_INITIAL_STATE:
  [
    -0.025876251,
    0.0,
    0.0,
    0.0,
    0.0,
    0.0,
    0.0,
    0.165,
    0.0,
    0.0,
    0.0,
    0.0,
    0.0,
    0.0,
  ]

KF_INITIAL_STATE_COVAR:
  [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
KF_INITIAL_STATE_COVAR_COEFF: 0.02
KF_PROC_COVAR:
  [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 20, 25, 20, 1.0, 1.0, 5.0]
KF_PROC_COVAR_COEFF: 0.002
KF_OBS_COVAR: [
    0.1785,
    -0.0755,
    -0.0262,
    0.0160,
    -0.0059,
    -0.0069,
    -0.0755,
    0.1212,
    0.0720,
    -0.0038,
    0.0049,
    0.0052,
    -0.0262,
    0.0720,
    0.2684,
    0.0033,
    0.0011,
    0.0108,
    0.0160,
    -0.0038,
    0.0033,
    0.0158,
    -0.0018,
    0.0013,
    -0.0059,
    0.0049,
    0.0011,
    -0.0018,
    0.0023,
    0.0008,
    -0.0069,
    0.0052,
    0.0108,
    0.0013,
    0.0008,
    0.0082,
  ] # For this one the user will define the whole 6x6 matrix. These are precalibrated using the model as groundtruth. 
KF_OBS_COVAR_COEFF: 1.0
